github/github.go

```diff
@@ -427,10 +443,18 @@ func CheckResponse(r *http.Response) error {
 	if err == nil && data != nil {
```

Comment:
Maybe it's worth expanding documentation of `CheckError` to mention both `TwoFactorAuthError ` and `RateLimitError` error types?